_posts/2016-04-28-hack-camp-is-back.md

```diff
@@ -0,0 +1,52 @@
+---
+layout:     post
+title:      Hack Camp Is Back
+date:       2016-04-28
+author:     Max Wofford
+summary:    We'll be running a Hack Camp this summer.
+---
+
+We're very excited to announce that Hack Camp, our two-week high school summer
+camp in downtown San Francisco, will be coming back this summer.
+
+There will be four sessions starting June 6th and ending August 19th. It's
+targeted towards complete beginners and we'll be covering the fundamentals of
+web development (i.e. building websites and web games). Find out more and apply
+at [https://camp.hackclub.com/](https://camp.hackclub.com/).
+
+![Hack Camp][hackcamp logo]{: .center-image}
+
+I could go on all day about how much of a blast it was last summer, but I
+figured last year's hackers can talk about it better than I can.
+
+> I felt euphoric at the end of each day because I had managed to implement a
+> feature I had previously thought impossible to implement. It really taught me
+> that I could 'distort reality' and make things HAPPEN.
+
+> The way Hack Camp is taught is totally opposite of what our school system is.
+> This camp really opened my mind to the different ways that I can learn.
+
+>  In the camp's mere two weeks, more than in any part of my life, not in my
+>  days as a fetus, an infant, a toddler, an elementary schooler, a middle
+>  schooler, a high school freshman, a high school sophomore, or a high school
+>  junior, I have never made so many mistakes.
+
+Here are a few pictures from last year:
+
+![Group photo with Tom Preston-Werner][hackcamp group photo]{: .half}
+![Group photo of our third session][hackcamp]{: .half}
+![Hackers working on a post-it littered table][hackcamp brainstorm]{: .half}
+![Hackers pretending to be cats][hackcamp energizer]{: .half}
+
+If you have any questions, please don't hesitate to reach out! We can be
+contacted at [summer@hackclub.com](mailto:summer@hackclub.com).
+
+Happy hacking!
+
+\- Max
+
+[hackcamp logo]: /assets/hackcamp_logo.svg "Thanks for making the logo Jessica!"
+[hackcamp group photo]: /assets/hackcamp_group_photo.jpg "Tom Preston-Werner showed up"
+[hackcamp]: /assets/hackcamp.jpg "Fantastic group picture with our third session"
+[hackcamp brainstorm]: /assets/hackcamp_brainstorming.jpg "Brainstorming with post-its"
+[hackcamp energizer]: /assets/hackcamp_energizer.jpg "I think this was our best impression of a cat"
```

Comment:
Use underscores in these types of links. There should also be space between "hack" and "camp".

- `[hack_camp_logo]
- `[hack_camp_group_photo]`
- etc